### PR TITLE
Bind Mac2Element in element map for MAC platform

### PR DESCRIPTION
**Pull request by DmitryMishtal**
## Change list

Bind MAC platform with Mac2Element to fix ClassCastException while Mac2Driver usage
 
## Types of changes

What types of changes are you proposing/introducing to Java client?
_Put an `x` in the boxes that apply_

- [ ] No changes in production code.
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Comment by mykola-mokhnach** [comment]
Thanks for fixing this
Could you please sign the CLA?

File: src/main/java/io/appium/java_client/internal/ElementMap.java [hunk 1]
```diff
@@ -37,7 +38,8 @@
     IOS_XCUI_TEST(AutomationName.IOS_XCUI_TEST.toLowerCase(), IOSElement.class),
     ANDROID_UI_AUTOMATOR(MobilePlatform.ANDROID.toLowerCase(), AndroidElement.class),
     IOS_UI_AUTOMATION(MobilePlatform.IOS.toLowerCase(), IOSElement.class),
-    WINDOWS(MobilePlatform.WINDOWS.toLowerCase(), WindowsElement.class);
+    WINDOWS(MobilePlatform.WINDOWS.toLowerCase(), WindowsElement.class),
+    MAC(MobilePlatform.MAC.toLowerCase().toLowerCase(), Mac2Element.class);
```
Comment (valfirst) [inline src/main/java/io/appium/java_client/internal/ElementMap.java:42]: Duplicate `toLowerCase` invocation

**Review by valfirst — CHANGES_REQUESTED** [review]

File: src/main/java/io/appium/java_client/internal/ElementMap.java (hunk 1 above)
Comment (DmitryMishtal) [reply src/main/java/io/appium/java_client/internal/ElementMap.java]: Indeed... Seems like fixed, sry :)

**Review by valfirst — APPROVED** [review]

**Comment by DmitryMishtal** [comment]
> Just make sure the email address of the corresponding local GH account matches to the one used to sign the CLA. The bot uses email value for comparison

Seems like i have two different emails in git config & git hub account... Should i close this PR and create a new one with correct email? Because as i understood i cant re-sign CLA for this PR with another email address

**Pull request closed without merging**